File: FE/src/types/UC_UseCaseDemo/UC_UseCaseDemo.ts
import { EntityType, SearchBase } from "../general";

export interface UC_UseCaseDemoType extends EntityType {
    idDuAn?: string;
    tenUseCase?: string;
    tacNhanChinh?: string;
    tacNhanPhu?: string;
    doPhucTap?: string;
    lstHanhDong?: string;
    lstHanhDongNangCao?: string;
    createdDate?: string;
}

export interface UC_UseCaseDemoCreateType  {
    idDuAn: string;
    tenUseCase: string;
    tacNhanChinh: string;
    tacNhanPhu?: string;
    doPhucTap: string;
    lstHanhDong?: string;
    lstHanhDongNangCao?: string;
    loaiUseCaseCode?: string;
}

export interface UC_UseCaseDemoEditType extends UC_UseCaseDemoCreateType {
    id: string;
}

export interface UC_UseCaseDemoSearchType extends SearchBase {
    tenUseCase?: string;
    tacNhanChinh?: string;
    tacNhanPhu?: string;
    doPhucTap?: string;
    idDuAn?: string;
} 

File: BE/Hinet.Model/Entities/DA_Test_Case/UC_UseCaseDemo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hinet.Model.Entities.DA_Test_Case
{
    public class UC_UseCaseDemo : AuditableEntity
    {
        public Guid IdDuAn { get; set; }
        public string? TenUseCase { get; set; }
        public string? TacNhanChinh { get; set; }
        public string? TacNhanPhu { get; set; }
        public string? DoPhucTap { get; set; }
        public string? loaiUseCaseCode { get; set; }
        public string? lstHanhDong { get; set; }
        public string? lstHanhDongNangCao { get; set; }
    }
}
